### PR TITLE
Add Gobbler.app v2.0

### PR DESCRIPTION
**Pull request by thehunmonkgroup**
https://www.gobbler.com
https://app.gobbler.com

### CODE REVIEW

**Comment by vitorgalvao** [comment]
[`installer :manual`](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#installer-manual) is what you’re looking for.

`uninstall` will also need to be added. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that.

**Comment by thehunmonkgroup** [comment]
Ok, I added the installer :manual stanza, and, I'm a bit lost on how to do the uninstaller part. I looked through the documentation, and nothing seemed to fit this use case -- there's no uninstall script bundled with their installer app that I can see, just the binary installer. It's not a package, and the only way I know how to uninstall is through their UI as I describe in my original commit.

Thoughts?

File: Casks/gobbler.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'gobbler' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://app.gobbler.com/download_latest_redirect?product_version=2'
+  name 'Gobbler'
+  homepage 'https://app.gobbler.com'
+  license :gratis
+
+  app 'Gobbler.app'
```
Comment (vitorgalvao) [inline Casks/gobbler.rb]: This needs to be removed.

**Comment by vitorgalvao** [comment]
You need to find how the app is correctly uninstalled, by asking the developer or searching on the website. Usually these situations have CLI instructions.

File: Casks/gobbler.rb (hunk 1 above)
Comment (thehunmonkgroup) [inline Casks/gobbler.rb]: I'm confused. What is the 'this' that needs to be removed? That looks like almost the entire cask file.

**Comment by thehunmonkgroup** [comment]
Ok, I think I found what I need for uninstall. The installer app installs 'Gobbler2 Uninstaller.app' in /Applications. That handles all of the uninstall including uninstalling itself. What's the proper way to run that via these cask uninstall options?

**Comment by thehunmonkgroup** [comment]
Ok, I think I found what I need for uninstall. The installer app installs 'Gobbler2 Uninstaller.app' in /Applications. That handles all of the uninstall including uninstalling itself. What's the proper way to run that via these cask uninstall options?

**Comment by vitorgalvao** [comment]
> What's the proper way to run that via these cask uninstall options?

There is none. We don’t support graphical uninstallers.

**Comment by thehunmonkgroup** [comment]
Ok, so what do you advise? Is there a way to get this cask to state that's commitable to the main repository?

**Comment by vitorgalvao** [comment]
Only after `uninstall` is done. You can try asking the developers, or try to figure out what the uninstaller does in a manner that is reproducible via CLI, and do that in the cask.

**Comment by thehunmonkgroup** [comment]
Ok, I dug through the code and found their uninstall script, committed an uninstall stanza and pushed.
Here's output from 'brew cask uninstall gobbler' -- seems to do the job:

```
brew cask uninstall gobbler
==> Removing App symlink: '/Users/hunmonk/Applications/Gobbler.app'
==> Running uninstall process for gobbler; your password may be necessary
==> Forgot package 'com.gobbler.GobblerDownstreamProxy' on '/'.
==> keychain: "/Users/hunmonk/Library/Keychains/login.keychain"
==> class: "genp"
==> attributes:
==>     0x00000007 <blob>="com.Gobbler.Gobbler2"
==>     0x00000008 <blob>=<NULL>
==>     "acct"<blob>=<NULL>
==>     "cdat"<timedate>=0x32303135303532353233313334355A00  "20150525231345Z\000"
==>     "crtr"<uint32>=<NULL>
==>     "cusi"<sint32>=<NULL>
==>     "desc"<blob>=<NULL>
==>     "gena"<blob>="com.Gobbler.Gobbler2.password"
==>     "icmt"<blob>=<NULL>
==>     "invi"<sint32>=<NULL>
==>     "mdat"<timedate>=0x32303135303532353233313334355A00  "20150525231345Z\000"
==>     "nega"<sint32>=<NULL>
==>     "prot"<blob>=<NULL>
==>     "scrp"<sint32>=<NULL>
==>     "svce"<blob>="com.Gobbler.Gobbler2"
==>     "type"<uint32>=<NULL>
==> No receipt for 'com.gobbler.Gobbler2' found at '/'.
==> 2015-05-25 18:15:31.663 defaults[16418:1389304] 
==> Domain com.Gobbler.Gobbler2 does not exist
==> password has been deleted.
```

The only remnant I see is a $HOME/Gobbler directory that the installer creates. I'm hesitant to blindly delete this in the case that the user may have added files there.

Is there anything else this Cask needs?

**Comment by fanquake** [comment]
Can you please squash your commits

**Comment by thehunmonkgroup** [comment]
Squashed and pushed. Please let me know if there's anything else you need before you merge.

File: Casks/gobbler.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/gobbler.rb]: I’m referring to the `app` line. It shouldn’t be there, since linking it wouldn’t make sense.

**Comment by vitorgalvao** [comment]
@fanquake Squashing is hard for many users; we shouldn’t be asking for that prematurely. There are still comments in this cask that need to be addressed (the `app` line).

**Comment by thehunmonkgroup** [comment]
Removed app line, retested install/uninstall, squashed commits, pushed. Anything else?

**Comment by vitorgalvao** [comment]
Everything looks great, @thehunmonkgroup. Thank you for all your work and patience on this. Merged.

**Comment by thehunmonkgroup** [comment]
fantastic! glad to help out.

**Pull request merged**